src/frameworks/sveltekit/index.ts

```diff
@@ -11,7 +11,8 @@ export const name = "SvelteKit";
 export const support = SupportLevel.Experimental;
 export const type = FrameworkType.MetaFramework;
 export const discover = viteDiscoverWithNpmDependency("@sveltejs/kit");
-export { getDevModeHandle } from "../vite";
+
+export { getDevModeHandle, supportedRange } from "../vite";
 
 export async function build(root: string) {
```

Comment:
sveltekit should probably have it's own supported range

Reply:
indeed but maybe after 2.0 is released? It seems like it won't be released anytime soon: https://github.com/sveltejs/kit/milestone/5

Reply:
Fair enough